### PR TITLE
LPD-31452/LPD-31453 Tests for sending current user email notifications after creating and updating a Object Entry

### PR DESCRIPTION
**Pull request by victorKammerer**
[LPD-31452 (Update)](https://liferay.atlassian.net/browse/LPD-31452)
[LPD-31453 (Create)](https://liferay.atlassian.net/browse/LPD-31453)

### CODE REVIEW

**Comment by victorKammerer** [comment]
ci:test:relevant

**Comment by victorKammerer** [comment]
ci:test:object

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java [hunk 1]
```diff
@@ -797,6 +804,108 @@ public void testSendNotificationWithRoles() throws Exception {
...
+				TestPropsValues.getUserId(),
+				objectDefinition.getObjectDefinitionId());
+
+		_user = _addUser();
+
+		Role role = _addRoleUserWithPermission(
+			new String[] {ObjectActionKeys.ADD_OBJECT_ENTRY}, objectDefinition,
+			_user);
+
+		_addResourcePermission(ActionKeys.UPDATE, objectDefinition, role);
```
Comment (linolaoj) [inline modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java:876]: Hi @victorKammerer, this method is used only once, can we make it inline?

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Comment by victorKammerer** [comment]
Done, @linolaoj!

**Comment by victorKammerer** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:forward

**Review by linolaoj — APPROVED** [review]

**Pull request closed without merging**